packages/ckeditor5-ui/tests/toolbar/toolbarview.js

```diff
@@ -526,117 +526,164 @@ describe( 'ToolbarView', () => {
 			factory.add( 'baz', namedFactory( 'baz' ) );
 		} );
 
-		it( 'expands the config into collection', () => {
-			view.fillFromConfig( [ 'foo', '-', 'bar', '|', 'foo' ], factory );
+		it( 'should use #createItemsFromConfig() and pass the result to #items', () => {
```

Comment:
For the record, the whole ugly changeset comes down to two things:
* moving all `fillFromConfig()` tests to `createItemsFromConfig()` section (with a slight adaptation),
* a new test for `fillFromConfig()` that verifies it became a wrapper around `createItemsFromConfig()`,